prime/src/data/actions/ActionOperators.ts

```diff
@@ -116,6 +117,10 @@ export function addLiquidityOperator(
     uniswapPositions.push({ liquidity, lower: lowerTick, upper: upperTick });
   }
 
+  if (uniswapPositions.length > MAX_UNISWAP_POSITIONS) {
+    throw Error(`Too many uniswap positions`);
+  }
+
```

Comment:
Why'd you move this here? (was in `runWithChecks` before)

Reply:
I thought it would be better to fail fast with this one to hopefully make it clear when this is the thing causing the errors.

Reply:
We could add an additional sanity check to runWithChecks, but I do like having this here.

Reply:
I think putting this one in here begs the question; why aren't all the others done here too -- just feels inconsistent.